zokrates/gadgets/pedersenHasher.py

```diff
@@ -0,0 +1,217 @@
+import math
+import bitstring
+from math import floor, log2
+from struct import pack
+
+from ..babyjubjub import Point, JUBJUB_L, JUBJUB_C
+from ..field import FQ
+
+WINDOW_SIZE_BITS = 2
```

Comment:
Consider adding a comment explaining what this parameter is. For example, I'm a bit confused, as when I think about the window size, I associate it with either the number of bits per window (186/189 depending on babyjujub/jubjub) or the 3-bit lookups. So, I'm not sure what it refers to here. I guess it could be beneficial to clarify.